### ISSUE TITLE
Uninstaller deletes the sia node

### ISSUE THREAD

**Issue by jkawamoto** [GooBox/goobox-community-gui#77]
Currently, the uninstaller keeps the sia node for debugging. At some point, it should be deleted.

**Comment by MeijeSibbel** [comment]
This is tricky though. Say someone uninstalls the app so that they can install a new version, they would lose their entire node + funds. uninstalling the app before installing a new one is a quite common practice for many users. Maybe we should add a tick box to the uninstaller to ask if they want to remove their full node and all files (with the waring that they will lose all files uploaded on that PC).

**Comment by jkawamoto** [comment]
True. At least, until sia supports seed based file recovery, we needs to provide an option to not delete contractor journals.
